.nvmrc

```diff
@@ -0,0 +1 @@
+4.2.6
```

Comment:
i'm not saying that the file isn't helpful.  i'm saying that because it falls into the category of personal preference (developers may or may not use nvm, they may or may not want to target `4.x`) it makes more sense to me for you to continue to use it locally and have `.gitignore` ensure its not checked into version control.